packages/legend-graph/src/graph-manager/AbstractPureGraphManager.ts

```diff
@@ -582,6 +587,10 @@ export abstract class AbstractPureGraphManager {
   abstract getQueryInfo(queryId: string): Promise<QueryInfo>;
   abstract createQuery(query: Query, graph: PureModel): Promise<Query>;
   abstract updateQuery(query: Query, graph: PureModel): Promise<Query>;
+  abstract updateQuerySelectedFields(
```

Comment:
Should we update this to be `patchQuery` to match the `legend-engine` change?

Reply:
thanks missed this one